grin/test_prim_ops.c

```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "prim_ops.h"
+
+// Compile with
+// clang-7 prim_ops.c test_prim_ops.c -o test_prim_ops
```

Comment:
where is the file for the expected output?

Reply:
But why don't we integrate this to the test system?
We can include C files now and we could compare with the expected stdout.